src/desktop/noo_frame.cpp

```diff
@@ -34,7 +34,7 @@ EVT_MENU(wxID_EXIT, NooFrame::exit)
 EVT_CLOSE(NooFrame::stop)
 wxEND_EVENT_TABLE()
 
-NooFrame::NooFrame(Emulator *emulator): wxFrame(nullptr, wxID_ANY, "NooDS"), emulator(emulator)
+NooFrame::NooFrame(Emulator *emulator): wxFrame(nullptr, wxID_ANY, "NooDS"), emulator(emulator), coreThread(NULL)
```

Comment:
I'd prefer doing this in the header file instead; there's no reason to use an initializer list to set it to a constant. Also, use nullptr instead of NULL, for consistency with the rest of the code.